src/compile/data/parse.ts

```diff
@@ -119,7 +119,7 @@ function parseRoot(model: Model, sources: SourceNode[]): DataFlowNode {
 
     if (model.data === null) {
       // data: null means we should ignore the parent's data so we just create a new data source
-      const source = new SourceNode([]);
+      const source = new SourceNode({values: []});
```

Comment:
I don't see any change to `SourceNode` constructor, is this going to break?

Reply:
`[]` is a not a valid `Data` (the expected type). I don't know why the code accepted an empty array before.